pgx-tests/src/tests/spi_tests.rs

```diff
@@ -306,19 +313,39 @@ mod tests {
             assert!(!a.is_empty());
             assert_eq!(1, a.len());
             assert!(a.get_heap_tuple().is_some());
-            assert_eq!(Some(1), a.get_datum(1));
+            assert_eq!(1, a.get_datum::<i32>(1).expect("a.get_datum::<i32>(1) failed").unwrap());
         });
     }
 
     #[pg_test]
+    #[ignore = "come back to this test"]
```

Comment:
I don't understand why this test fails.  I guess I don't understand what it's testing

Reply:
It's testing that we longer have a bug in that we always use `SPI_processed` (#938)

Reply:
My bad, it was definitely missing a comment.
I was thinking something along these lines:

```rust
    #[pg_test]
    fn test_open_multiple_tuptables() {
        // Regression test to ensure a new `SpiTupTable` instance does not override the
        // effective length of an already open one due to misuse of Spi statics
```

```rust
    #[pg_test]
    #[ignore = "come back to this test"]
    fn test_open_multiple_tuptables_rev() {
        // Regression test to ensure a new `SpiTupTable` instance does not override the
        // effective length of an already open one.
        // Same as `test_open_multiple_tuptables`, but with the second tuptable being empty  
```

Reply:
Thanks.  I've pushed a few minor changes to these two tests in bc27ff31cd253855163fc066228214eda0bb912a.

Reply:
ha!  and 9074043891d63cb2d660d650af79dcdcfa03e394.